protocols/bgp/server/server.go

```diff
@@ -45,10 +55,15 @@ type BGPServer interface {
 
 // NewBGPServer creates a new instance of bgpServer
 func NewBGPServer(routerID uint32, defaultVRF *vrf.VRF, listenAddrsByVRF map[string][]string) BGPServer {
-	return newBGPServer(routerID, defaultVRF, listenAddrsByVRF)
+	return newBGPServer(routerID, defaultVRF, listenAddrsByVRF, DefaultBGPServerConfig)
```

Comment:
If we start using a ServerConfig structure shouldn't that contain the routerID and maybe even the listening addresses too?

Reply:
Fine with me, will refactor